Tests/SonarScanner.MSBuild.PreProcessor.Test/EmbeddedAnalyzerInstallerTests.cs

```diff
@@ -95,6 +80,7 @@ public void EmbeddedInstall_SinglePlugin_SingleResource_Succeeds()
             var actualFiles = testSubject.InstallAssemblies(new Plugin[] { requestedPlugin });
 
             // Assert
+            logger.AssertInfoLogged("Processing plugin: plugin1 version 1.0");
```

Comment:
Assertion moved from `InstallAssemblies_ValidPlugins_PluginsVersionLogInfo`.
The previous test was not feasible anymore due to the introduction of the new exception being thrown when plugins are missing.